doc/classes/AnimationNodeStateMachinePlayback.xml

```diff
@@ -6,10 +6,16 @@
 	<description>
 		Allows control of [AnimationTree] state machines created with [AnimationNodeStateMachine]. Retrieve with [code]$AnimationTree.get("parameters/playback")[/code].
```

Comment:
Maybe remove the "Retrieve with .." line, since the codeblock shows how to get it. (Or just remove the `code` tag)